e2e/firstTest.spec.js

```diff
@@ -298,7 +298,7 @@ describe('ReactNative Support', () => {
   describe('Autotrack', () => {
     it("should autotrack 'TouchableOpacity's", async () => {
       const expectedHierarchy =
-        'AppContainer;|App;|Provider;|Connect(MainScreen);|MainScreen;|TouchableOpacity;|';
+        'AppContainer;|App;|Provider;|NavigationContainer;|Navigator;|NavigationView;|TabNavigationView;|ScreenContainer;|ResourceSavingScene;|SceneView;|Connect(TouchablesPage);|TouchablesPage;|TouchableOpacity;|';
```

Comment:
Wow that's a lot of added hierarchy for wrapping this app in a tab navigator! O_O

Wonder how soon we're going to have to start adding easy ways to filter out noise in these / simplify them? I suppose the length doesn't matter if they're stable and you're using the right tools to create the defs.

Reply:
Yeah, that is something I still don't have the best understanding of.  This seems similar to CSS hierarchies in terms of complexity, no?

Reply:
Lots of CSS hierarchies are much worse for sure. I think this was the first time I saw a real tab nav hierarchy so the length surprised me.